src/Sylius/Bundle/UiBundle/Resources/private/js/semantic-ui-actions.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+(function ($) {
+    'use strict';
+    
+    $.fn.api.settings.api = {
+        'get taxons' : '/admin/api/taxons',
```

Comment:
I'm not convinced. "Get taxons" explains exactly what happend when you use 'admin/api/taxons'.